package.json

```diff
@@ -20,18 +20,20 @@
     "freedom": "^0.6.18"
   },
   "devDependencies": {
-    "es6-promise": "^1.0.0",
     "arraybuffer-slice": "~0.1.2",
+    "browserify-istanbul": "^0.1.3",
+    "es6-promise": "^1.0.0",
     "freedom": "^0.6.11",
-    "grunt": "^0.4.5",
-    "grunt-contrib-jasmine": "^0.8.0",
-    "grunt-contrib-clean": "^0.6.0",
-    "grunt-contrib-coffee": "^0.12.0",
-    "grunt-contrib-copy": "^0.7.0",
-    "grunt-contrib-symlink": "^0.3.0",
-    "grunt-contrib-uglify": "^0.7.0",
-    "grunt-contrib-watch": "^0.6.1",
+    "grunt": "~0.4.5",
     "grunt-browserify": "^3.0.1",
+    "grunt-contrib-clean": "~0.6.0",
```

Comment:
Sorting is good but why change these back to tildes? This creates merge conflicts unnecessarily later...